src/components/articleItems.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react"
+import { getAllArticles } from "./api"
+import { ArticleBox } from "./articleBox"
+
+export const ArticleItems = () => {
+    const [isLoading, setIsLoading] = useState(false);
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        setIsLoading(true);
+        getAllArticles().then((response) => {
+            setIsLoading(false);
+            if (response && response.articles) {
+                setArticles(response.articles);
+            } else {
+                setArticles([])
+            }
+        }).catch((error) => {
+            console.error("Error fetching articles: ", error);
+            setIsLoading(false)
+        })
+},[])
+
+    return (
+        <section>
+            <div className="articles-box-style">
+                {isLoading ? (
+                    <p>Loading...</p>
+                ) : (
+                    <ul className="article-format">
+                        {articles && articles.length > 0 ? (
```

Comment:
If your loading state is false the articles exist as defined in your useEffect. As you're only setting the loading state after the api call resolves.